schemas/transaction/getTrensactionSchema.js

```diff
@@ -0,0 +1,27 @@
+const { ErrorObject } = require("../../helpers/error");
+
+const idSchema = {
+  id: {
+    in: ["params"],
+    isInt: true,
+    toInt: true,
+    errorMessage: "ID is wrong",
+  },
+  custom: {
+    options: async (req) => {
+      try {
+        // const user = await Users.findByPk(userId);
+        // if (!user) throw new ErrorObject("User not found", 404);
+        const { id } = req.params;
+        const response = await Transactions.findByPk(id);
```

Comment:
add transaction model so that the custom can work:

`const { Transactions } = require('../../database/models');`

testing with an id that does not exist throws status 200 and should throw an error:

![Captura de Pantalla 2022-11-06 a la(s) 10 13 54 p m](https://user-images.githubusercontent.com/84211273/200218903-06747b84-03c3-4b9d-b050-6030dacf0e05.png)